deny.toml

```diff
@@ -60,30 +60,6 @@ expression = "LicenseRef-ring"
 version = "0.16.20"
 license-files = [{ path = "LICENSE", hash = 0xbd0eed23 }]
 
-# TODO: remove this if / once there is an SPDX entry for ELv2
-[[licenses.clarify]]
-name = "apollo-router"
-expression = "LicenseRef-ELv2"
-version = "0.1.0-preview.7"
-license-files = [{ path = "LICENSE", hash = 0xaceadac9 }]
-
-[[licenses.clarify]]
-name = "apollo-router-core"
-expression = "LicenseRef-ELv2"
-version = "0.1.0-preview.7"
-license-files = [{ path = "LICENSE", hash = 0xaceadac9 }]
-
-[[licenses.clarify]]
-name = "router-bridge"
-expression = "LicenseRef-ELv2"
-license-files = [{ path = "router-bridge/LICENSE", hash = 0xaceadac9 }]
-
```

Comment:
I've also opened https://github.com/apollographql/federation-rs/pull/118 to apply a similar license treatment to the `router-bridge` itself, thus enabling this to go away and not be a `warn` going forward.